### DISCUSSION TITLE
Save as png with custom dimensions

### DISCUSSION THREAD

**Discussion by Alphazork** [razztyfication/vue-drawing-canvas#18] [Ideas]
Is there a way to save the canvas as a png but with smaller proportions?
Maybe give the save(...) function width and height parameters and save it that way?

**Comment by razztyfication** [comment]
good idea, maybe I'll give it a try.

hey I have a question, do you mind using any depedencies when using it? I am planning rebuilt it with vite and composition api but it only support vue 3. Maybe I'll release it with v2.0.0 with some breaking changes

**Reply by Alphazork** [reply]
I don't mind as long as it works.
Speed could be improved, especially with the scale set at 2x like I described a month ago
